dbms/tests/queries/0_stateless/00387_use_client_time_zone.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+TZ=UTC clickhouse --client --use_client_time_zone=1 --query="SELECT toDateTime(1000000000)"
```

Comment:
How are these tests usually launched? Obviously one would want to test with a freshly compiled client and this requires manually setting $PATH variable.